boards/cc3200-launchxl/dist/cc3200.cfg

```diff
@@ -0,0 +1,8 @@
+source [find board/ti-cc3200-launchxl.cfg]
+$_TARGETNAME configure -rtos auto
+
+$_TARGETNAME configure -event gdb-attach {
+# cc3200_dbginit $_TARGETNAME
+# cortex_m dbginit
```

Comment:
are those commented out?

Reply:
since I switched to OpenOCD provided config those two lines are no longer requires. I will remove them to reduce noise.